desktop/libs/notebook/src/notebook/connectors/flink_sql.py

```diff
@@ -162,26 +162,30 @@ def _get_session(self):
     session = self._get_session_info_from_user()
 
     if not session:
-      session = self.db.create_session()
-      if self.default_database:
-        self._use_database(self.default_catalog, self.default_database)
-      elif self.default_catalog:
-        self._use_catalog(self.default_catalog)
-
+      session = self._create_session_and_use_default_database()
     try:
       self.db.session_heartbeat(session_handle=session['sessionHandle'])
     except Exception as e:
       if "Session '%(sessionHandle)s' does not exist" % session in str(e):
         LOG.warning('Session %(sessionHandle)s does not exist, opening a new one' % session)
-        session = self.db.create_session()
+        session = self._create_session_and_use_default_database()
       else:
         raise e
 
-    session['id'] = session['sessionHandle']
     self._set_session_info_to_user(session)
 
     return session
 
+  def _create_session_and_use_default_database(self):
```

Comment:
Can you use _create_session as the function name instead of the above one?

Reply:
done